presets/4.4/tune/DavideFPV_DRock_3_Tune.txt

```diff
@@ -0,0 +1,192 @@
+#$ TITLE: Davide FPV D-Rock 3 Tune
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: DavideFPV, Davide FPV, 6S, Freestyle, 5 inch, 5", Davide, Drock, D-ROCK, v3, drock v3, D-rock v3, drock 3, d-rock 3
+#$ AUTHOR: Davide FPV (Davide Angeletti)
+
+#$ PARSER: MARKED
+#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
+
+#$ DESCRIPTION: Davide FPV (Davide Angeletti) is a Professional Italian Pilot and Vocational Trainer, Youtube content creator and professional tuner. YT Channel:  [https://www.youtube.com/davidefpv](https://www.youtube.com/davidefpv) IG and TikTok: @davidefpv
+#$ DESCRIPTION:
+#$ DESCRIPTION: Information:
+#$ DESCRIPTION: -----------
+#$ DESCRIPTION: Davide FPV TUNING for the D-ROCK 3 build with 6S, HQProp J40 and 1750-1850kv range motors. 
+#$ DESCRIPTION: -----------
+#$ INCLUDE: presets/4.4/tune/defaults.txt
+
+# -- Antigravity --
+
+set anti_gravity_gain = 115
+set craft_name = UNSTOPPABLE
+set throttle_boost = 0
+
+#$ OPTION_GROUP BEGIN:PIDs
+
+    #$ OPTION BEGIN (CHECKED): 6S Lipo with HD Camera (1750-1850kv motors)
+
+	set iterm_relax_cutoff = 13
+	set simplified_master_multiplier = 125
```

Comment:
in order for sliders to take effect you need to put
"simplified_tuning apply" at the end of a preset file
otherwise, this and all the following sliders commands will only move sliders, but not the actual values.

Reply:
Thanks for reviewing. I'll correct everything and add this tune in another branch with relative pull request.